lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -132,21 +145,32 @@ public Map<String, Object> handleRequest(JourneyRequest event, Context context)
                 return JOURNEY_VCS_NOT_CORRELATED.toObjectMap();
             }
 
-            boolean hasMatchingGpg45Profile =
-                    hasMatchingGpg45Profile(
+            var contraIndicators =
+                    ciMitService.getContraIndicators(userId, govukSigninJourneyId, ipAddress);
```

Comment:
At the moment yes, I think it would be safe to skip here because we'd have failed scoring earlier in the journey. But is that something we want to rely on remaining true in the future, and would we notice if it broke?